packages/documentation-framework/components/gdprBanner/gdprBanner.js

```diff
@@ -13,7 +13,7 @@ export const GdprBanner = () => {
   }
 
   return !isBannerOpen ? null : (
-    <div className="ws-gdpr-banner-container pf-v5-l-flex">
+    <section aria-labelledby="ws-gdpr-banner-text" className="ws-gdpr-banner-container pf-v5-l-flex">
```

Comment:
Nit: this can cause a rather verbose output for the section's accessible name when traversing the rotor menu for something like VO or JAWS. Wondering if instead of `aria-labelledby` if `aria-label` would be better for a more succinct label. "Cookies on PatternFly" or something. Probably not a blocker, though, since it does provide the info to the user with less potential to "miss it" (outside of literally skipping past it before a user's AT can announce it fully)

Reply:
Thanks for the clarification, that makes sense - I will adopt your suggestion